src/main/features/dashboard/views/defendant.njk

```diff
@@ -21,6 +21,18 @@
               { postProcess: 'sprintf', sprintf: [ 'https://www.gov.uk/county-court-judgments-ccj-for-debt/if-you-dont-owe-the-money' ]}) | safe }}</p>
       {% else %}
         <p>{{ t('Your defence is being processed. We will email you to explain what to do next.') }}</p>
+        <h2 class="heading-medium">{{ t('More options') }}</h2>
+        <h2 class="heading-medium">{{ t('Settle out of court') }}</h2>
+        <p>{{ t('You can still <a href="%s">make an offer</a> to settle the claim out of court. You will avoid a hearing and a County Court Judgment against you.',
+                { postProcess: 'sprintf', 
+                  sprintf: [ offerPageUri ] 
+                }) | safe 
+            }}
+        </p>
+
+        <h2 class="heading-medium">{{ t('Request mediation') }}</h2>
+        <p>{{ t('<a href="3">Request free telephone mediation</a>. Many people resolve their dispute before a hearing by taking part in a telephone mediation.') }}</p>
```

Comment:
why is this linked to 3?

Reply:
Is `Request mediation` option part of the offer story? Please double check with business.

Reply:
No, I will remove the link